INN/Largo
# Translatability fixes

## Changes

This pull request makes the following changes:

- reduces Largo to a single text domain: `largo`
- adds text domains to a whole heap of translation functions
- adds new translation to a few places
- removes some variables from translation strings, or removes the translation outright
- adds `translators:` notes for strings translated for use with `sprintf`/`printf`, based on the output of `grunt pot`

This PR does not include:
- an exhaustive search for cases where variables relevant to the translated text are appended to the translated text outside of the translation function, such as this example: `<php echo esc_html__( 'My name is', 'largo' ); . ' ' . $name ?>`

## Why



For #1592 and https://github.com/INN/umbrella-caribbean/issues/96

## Testing/Questions

Features that this PR affects:

- internationalization


Questions that need to be answered before merging:

- [ ] Are there other errors of translation that need to be fixed?

Steps to test this PR:

1. Run `grunt pot`. Expect no error messages.
2. Run the Theme Check plugin against Largo. Expect no translation-related error messages other than those noted below.

> WARNING: Found a translation function that is missing a text-domain. Function __, with the arguments 'largo'

I can't find this one. Good luck if you're going hunting.

> `$this->lib_e(`

The theme check plugin appears to think `lib_e` is a translation function. It's not; these results are a false positive.

> Possible variable $s found in translation function in inc/widgets/largo-twitter.php. Translation function calls must NOT contain PHP variables.
> 
> Line 34: __( '@%1$s's Likes on Twitter', 'largo' ),

That's the right format for passing a variable to sprintf, right?
